src/Tribe/Events/Views/V2/Models/Tickets.php

```diff
@@ -181,13 +181,18 @@ public function fetch_data() {
 
 						$number = number_format_i18n( $stock );
 
+						$ticket_label_singular = tribe_get_ticket_label_singular_lowercase( 'event-tickets' );
+						$ticket_label_plural = tribe_get_ticket_label_plural_lowercase( 'event-tickets' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space

<sub>hash: 114cf6b92327e683131b9fc4d94bf083</sub>